server/app/services/geo/esri/EsriServiceAreaValidationOption.java

```diff
@@ -50,13 +51,14 @@ public abstract static class Builder {
 
   /**
    * Determine if this EsriServiceAreaValidationOption is in the provided list of {@link
-   * ServiceAreaInclusion}.
+   * ServiceAreaInclusion}. Filters out FAILED states.
    *
    * @return boolean.
    */
   public Boolean isServiceAreaOptionInInclusionGroup(
       ImmutableList<ServiceAreaInclusion> inclusionGroup) {
     return inclusionGroup.stream()
+        .filter((inclusion) -> inclusion.getState() != ServiceAreaState.FAILED)
```

Comment:
**suggestion (blocking):** prefer `.equals` for non-primitives `ServiceAreaState.FAILED.equals(inclusion.getState())`

Reply:
Updated comparison.